src/test/java/com/connexta/store/service/impl/StoreServiceImplTest.java

```diff
@@ -338,15 +325,15 @@ void testGettingDatasetNotStored() {
     when(metacardStorageAdaptor.getStatus(eq(DATASET_ID))).thenReturn(QUARANTINED);
 
     assertThrows(RetrieveException.class, () -> storeService.getData(DATASET_ID, "metacard"));
-    verifyNoInteractions(indexDatasetClient);
+    verifyNoInteractions(indexClient);
   }
 
   @Test
   void testGettingDatasetNullStatus() {
     when(metacardStorageAdaptor.getStatus(eq(DATASET_ID))).thenReturn(null);
 
     assertThrows(RetrieveException.class, () -> storeService.getData(DATASET_ID, "metacard"));
-    verifyNoInteractions(indexDatasetClient);
+    verifyNoInteractions(indexClient);
```

Comment:
✏️ Move `verifyNoInteractions(indexClient)` to `@AfterEach` because this should be verified for every test method